docs/content/stable/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -74,22 +75,17 @@ on individually, which you should do in the following cases:
  to create indexes on each partition individually.
 * [Row-level geo-partitioning](../../../../../explore/multi-region-deployments/row-level-geo-partitioning/) is being used. In this case, create the index separately on each partition to
  customize the tablespace in which each index is created.
-* `CREATE INDEX CONCURRENTLY` is not supported for partitioned tables (see [CONCURRENTLY](#concurrently)).
 
 ### UNIQUE
 
 Enforce that duplicate values in a table are not allowed.
 
-### CONCURRENTLY
-Enable online schema migration (see [Semantics](#semantics) for details), with some restrictions:
-* Upon creating an index on a temporary table, online schema migration will be disabled.
-* `CREATE INDEX CONCURRENTLY` is not supported for partitioned table.
-* `CREATE INDEX CONCURRENTLY` is not supported inside a transaction block.
-
 ### NONCONCURRENTLY
 
 Disable online schema migration (see [Semantics](#semantics) for details).
 
+Concurrent is the default, but the grammar does not yet support `CONCURRENTLY`.
```

Comment:
Maybe we can just remove this line? We already said we don't support CONCURRENTLY keyword at line 46.

Reply:
It is removed in the latest version.